Applications/ConsoleReferenceSubscriber/Program.cs

```diff
@@ -206,9 +247,10 @@ private static void UaPubSubApplication_MetaDataDataReceived(object sender, Subs
         {
             lock (m_lock)
             {
+                Console.WriteLine("MetaDataDataReceived event:");
                 if (e.NetworkMessage is JsonNetworkMessage)
```

Comment:
Would it make sense to handle the UADP MetaData Message as well, like in the UaPubSubApplication_DataReceived()?

Reply:
@cristipogacean Yes, treating the UADP Metadata Message similar to the handling in UaPubSubApplication_DataReceived() is a good idea.